tools/iscc-sum/iscc_sum.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="iscc_sum" name="Generate ISCC-SUM" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>Generate content code and similarity hash from any file</description>
+    
+    <requirements>
+        <requirement type="package" version="0.1.0">iscc-sum</requirement>
+    </requirements>
+    
+    <version_command>iscc-sum --version</version_command>
+    
+    <command detect_errors="exit_code"><![CDATA[
+        iscc-sum '$input_file' | cut -d ' ' -f 1 > '${output_file}'
+    ]]></command>
+    
+    <inputs>
+        <param name="input_file" type="data" format="data" label="Input File" 
+               help="Any file type - ISCC-SUM will generate a checksum and similarity hash"/>
+    </inputs>
+    
+    <outputs>
+        <data name="output_file" format="txt" label="${tool.name} on ${on_string}"/>
+    </outputs>
+    
+    <tests>
+        <test>
+            <param name="input_file" value="test1.png"/>
+            <output name="output_file" file="test1_png.txt"/>
```

Comment:
```suggestion
            <output name="output_file">
                <assert_contents>
                    <has_line line="K4AOMGOGQJA4Y46PAC4YPPA63GKD5RVFPR7FU3I4OOEW44TYXNYOTMY" />
                </assert_contents>
            </output>
```

Reply:
similary for test2 and test 3, you can remove all the test output files and assert contents